app/mailers/user_mailer.rb

```diff
@@ -9,4 +11,16 @@ def welcome_email(user, password)
     @password = password
     mail(to: user.email, subject: 'Welcome to the Sara Alert system')
   end
+
+  def purge_notification
+    receipients = User.with_any_role(:public_health, :public_health_enroller)
+    @expiration_date = Chronic.parse(ADMIN_OPTIONS['weekly_purge_date']).strftime('%A %B %d, at %l:%M %p %Z')
+
+    receipients.each do |user|
+      @user = user
+      @num_purgeable_records = user.viewable_patients.purge_eligible.length
+
+      mail(to: user.email, subject: 'Sara Alert User Records Expiring Soon')
```

Comment:
What happens if `@num_purgeable_records == 0`? Probably want to `next`

Reply:
I was thinking about this case, and I wonder if the user will get worried if they don't receive the email one week and get concerned they don't know what is getting purged. For the time being I left it at sending an email with a zero. It may also be beneficial to have a separate email informing them that no patient data will be purged. If you think no need, I can add the `next`.

Reply:
That is a great point. Perhaps put a conditional into the email, stating no patients are eligible for purge.

Reply:
Updated to conditionally modify the message and subject to accommodate no purges on a given week.
![Screen Shot 2020-04-28 at 12 07 45 AM](https://user-images.githubusercontent.com/15388200/80446340-93188480-88e4-11ea-8635-8aa2a3931c25.png)